fhir-model/src/test/java/com/ibm/fhir/model/util/test/ReferencePatternTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.model.util.test;
+
+import static com.ibm.fhir.model.util.FHIRUtil.REFERENCE_PATTERN;
+
+import java.util.regex.Matcher;
+
+public class ReferencePatternTest {
+    public static void main(String[] args) throws Exception {
```

Comment:
Worthwhile making a Test since it'd be a good confirmation.

Reply:
Done